### PR TITLE
Add a 'Development' category to docmenu

### PR DESCRIPTION
**Pull request by jouvin**
Add a menu to easily locate the most important documentation for developers.

### CODE REVIEW

File: _includes/docmenu.md [hunk 1]
```diff
@@ -12,5 +12,9 @@
   * [Configuration Modules](/documentation/14.8.0/components/)
   * [Template Library](http://www.quattor.org/documentation/2014/06/06/how-to-use-template-library.html)
 
+##### Development
+  * [Basics](documentation/2016/03/08/development-basics.html)
```
Comment (jrha) [inline _includes/docmenu.md]: Needs a leading `/`

**Comment by jouvin** [comment]
retest this please

**Comment by jouvin** [comment]
test this please

**Comment by jouvin** [comment]
retest this please

**Comment by jouvin** [comment]
Typo fixed.

**Pull request merged**